### PR TITLE
Refactoring models

### PR DESCRIPTION
**Pull request by KostinP**
Models were in random folders (Models, ModelResponse), now they are in the correct folders (Models, Requests, Responses)
And FindNewsParams renamed to FindNewsFilter and moved to Requests.Filters

### CODE REVIEW

File: src/NewsService.Models.Dto/NewsService.Models.Dto.csproj [hunk 1]
```diff
@@ -2,4 +2,10 @@
 
   <Import Project="$(ProjectDir)..\..\Common.Configurations.targets" />
 
+  <ItemGroup>
+    <Compile Remove="Requesrs\**" />
+    <EmbeddedResource Remove="Requesrs\**" />
+    <None Remove="Requesrs\**" />
+  </ItemGroup>
```
Comment (djino97) [inline src/NewsService.Models.Dto/NewsService.Models.Dto.csproj:9]: What is this?

File: src/NewsService.Data/Interfaces/INewsRepository.cs [hunk 2]
```diff
@@ -35,6 +35,6 @@ public interface INewsRepository
         /// Find news in the database.
         /// </summary>
         /// <param name="findNewsParams">Params for serch.</param>
```
Comment (djino97) [inline src/NewsService.Data/Interfaces/INewsRepository.cs:37]: need fix

**Review by djino97 — CHANGES_REQUESTED** [review]

File: src/NewsService.Business/Interfaces/IFindNewsCommand.cs [hunk 3]
```diff
@@ -1,11 +1,11 @@
...
+﻿using LT.DigitalOffice.NewsService.Models.Dto.Requests.Filters;
+using LT.DigitalOffice.NewsService.Models.Dto.Responses;
 using System.Collections.Generic;
 
 namespace LT.DigitalOffice.NewsService.Business.Interfaces
 {
     public interface IFindNewsCommand
     {
-        public List<NewsResponse> Execute(FindNewsParams findNewsParams);
+        public List<NewsResponse> Execute(FindNewsFilter findNewsFilter);
```
Comment (BelehovEgor) [inline src/NewsService.Business/Interfaces/IFindNewsCommand.cs:9]: delete public

**Review by BelehovEgor — CHANGES_REQUESTED** [review]

File: src/NewsService.Models.Dto/NewsService.Models.Dto.csproj (hunk 1 above)
Comment (KostinP) [reply src/NewsService.Models.Dto/NewsService.Models.Dto.csproj]: Fixed.

File: src/NewsService.Data/Interfaces/INewsRepository.cs (hunk 2 above)
Comment (KostinP) [reply src/NewsService.Data/Interfaces/INewsRepository.cs]: Fixed.

File: src/NewsService.Business/Interfaces/IFindNewsCommand.cs (hunk 3 above)
Comment (KostinP) [reply src/NewsService.Business/Interfaces/IFindNewsCommand.cs]: Removed.

**Review by BelehovEgor — APPROVED** [review]

**Review by borovHokage — APPROVED** [review]

**Review by TIove — APPROVED** [review]

**Review by djino97 — APPROVED** [review]

**Pull request merged**